File: AULA_02/C/MediaPonderada.c
#include <stdio.h>

int main ( ){

  float x,y,w,media;

  printf("Entre com x, y e w: ");
  scanf("%f %f %f",&x,&y,&w);

  media = w*x + (1-w)*y;
  printf("A média é %f\n", media);

  return 0;
}

File: AULA_02/Haskell/MediaPonderada.hs
main = do
       putStrLn "Entre com x e y e w: "
       line <- getLine
       let x = read line
       line <- getLine
       let y = read line 
       line <- getLine
       let w = read line
       putStrLn $ "Média = " ++ show ( w*x + (1.0-w)*y )
